test/repository/like.repository.mock.ts

```diff
@@ -0,0 +1,5 @@
+export const MockLikeRepository = () => ({
+  getLikeByInput: jest.fn(),
```

Comment:
반환값 기본 타입은 명시해주시면 좋을 것 같습니다
```suggestion
  getLikeByInput: jest.fn().mockResolveValue(undefined),
```

Reply:
옙옙~ mock의 경우 테스트 통과를 위해서 일단 넣어놨는데 말씀주신것처럼 수정해놓겠습니다.